lib/chef_backup/helpers.rb

```diff
@@ -82,6 +82,7 @@ def ensure_file!(file, exception, message)
 
   def shell_out(*command)
     cmd = Mixlib::ShellOut.new(*command)
+    cmd.timeout ||= config['shell_out_timeout'] unless config['shell_out_timeout'].nil?
```

Comment:
In fact I think the way the command args for Mixlib::ShellOut.new works is that you can actually just pass it an options for `{'timeout' => config['shell_out_timeout']}` unless command seems to contain one itself.

So this might works better - I haven't looked at the callers of shell_out and don't know if they send in their own options but this will handle that.

```
options = command_args.last.is_a?(Hash) ? command_args.pop : {}
opts_with_defaults = { 'timeout' => config['shell_out_timeout'] }.merge(options)
cmd = Mixlib::ShellOut.new(*command, opts_with_defaults)
```